esp/scm/ws_dfu.ecm

```diff
@@ -158,7 +158,7 @@ ESPStruct [nil_remove] DFUFilePartsOnCluster
 ESPStruct [nil_remove] DFUFileProtect
 {
     string Owner;
-    int Count;
+    [depr_ver("1.53")] int Count;
```

Comment:
Should be [depr_ver("1.54")]. The depr_ver means 'not used from this version on'.

Reply:
ok will fix.